core/src/main/kotlin/com/expediagroup/sdk/core/model/exception/client/ExpediaGroupInvalidFieldNameException.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2022 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.sdk.core.model.exception.client
+
+/**
+ * Thrown to indicate that one or more passed field names are invalid.
+ *
+ * @param fieldsNames the names of the invalid fields.
+ */
+class ExpediaGroupInvalidFieldNameException(fieldsNames: Collection<String>) :
+    ExpediaGroupClientException("All fields names must contain only alphanumeric characters in addition to - and _ but found [${fieldsNames.joinToString(",")}]") {
```

Comment:
Can we have the error specifying the exact filed with an issue?

Reply:
The errors shows only the invalid fields names, meaning it filters out the valid ones.